### PR TITLE
Password Generator using Vanilla JS #14

### LINKED ISSUE #14: Password Generator using Vanilla JS
### WebMasterLog Repository (Proposing new issue)
:red_circle: **Project Title** : Password Generator
:red_circle: **Aim** : Website to generate password of different lengths and patterns for the user according to the user's convenience.
:red_circle: **Tech Stack Used** : HTML, CSS, JS
:red_circle: **Category** :  Vanilla-JS

***********************************************************************
### 📍 **Follow the Guidelines to Contribute in the Project :**
* You need to create a separate folder named as the Project Title.
* Inside that folder, there will be all the main components.
   * **Cover image** of the application, if project will be deployed in future, with the name of the folder in Kebab-Case (hypen-separated).
   * Main files in formatted state so that code will be readable.
   * Other Images - In seprate folder named `Images` used to form the application.
* Inside the `Project` folder, the `README.md` file must be filled up properly, with proper visualizations and descriptions.

***********************************************************************
:red_circle::yellow_circle: **Points to Note :**

- The issues will be assigned on a first come first serve basis, 1 Issue == 1 PR.
- "Issue Title" and "PR Title should be the same. Include issue number along with it.
- Follow [Contributing Guidelines](https://github.com/Avdhesh-Varshney/WebMasterLog/blob/main/CONTRIBUTING.md) & [Code of Conduct](https://github.com/Avdhesh-Varshney/WebMasterLog/blob/main/CODE_OF_CONDUCT.md) before start Contributing.

***********************************************************************
:white_check_mark: **To be Mentioned while taking the issue :**
- Full name : 
- GitHub Profile Link : 
- What is your participant role? (Mention the Open Source Program name. Eg. GSOC, GSSOC, SSOC, JWOC, etc.)

***********************************************************************
Happy Contributing 🚀 

All the best. Enjoy your open source journey ahead. 😎

### PR DESCRIPTION
**Pull request by Hacxmr**
# Pull Request for WebMasterLog 💡

## Issue Title: 
Password Generator using Vanilla JS #14

- **Name of the project** : Password Generator
- **Category:**  Vanilla Js
- **Name:** Mitali Raj
- **GitHub ID:** https://github.com/Hacxmr
- **Idenitfy yourself:** JWOC Contributor

closes #14

# Password Generator Web Page

## Overview
This HTML document creates a password generator web page. It features a heading, a display area for the generated password, and a button to trigger password generation. Users can also copy the generated password to the clipboard using an associated icon.

## Components
- Heading with a styled span for emphasis.
- Display area with an input field for the generated password.
- Copy icon for copying the password to the clipboard.
- "Generate Password" button.
- External CSS file (`style.css`) for styling.
- External JavaScript file (`script.js`) for handling functionality.

## Interaction
- Users can click the "Generate Password" button to generate a random password.
- The generated password can be copied to the clipboard using the copy icon.

## Note
- The actual functionality of generating and copying passwords is expected to be implemented in the linked JavaScript file (`script.js`).

## Checklist: ☑️

☑️My code follows the [Contributing Guidelines](https://github.com/Avdhesh-Varshney/WebMasterLog/blob/main/CONTRIBUTING.md) & [Code of Conduct](https://github.com/Avdhesh-Varshney/WebMasterLog/blob/main/CODE_OF_CONDUCT.md) of this project
☑️ This PR does not contain plagiarized content.
☑️ I have performed a self-review of my own code.
☑️ I have commented my code, particularly wherever it was hard to understand.
☑️ I have made corresponding changes to the documentation.
☑️ My changes generate no new warnings.
☑️ I have added things that prove my fix is effective or that my feature works.
☑️ Any dependent changes have been merged and published in downstream modules.

## Screenshots 📷
image

### CODE REVIEW

File: Vanilla-JS-Projects/Intermediate/Password Generator/Readme.md
Comment (Avdhesh-Varshney) [inline Vanilla-JS-Projects/Intermediate/Password Generator/Readme.md:1]: @Hacxmr Follow the readme template as mentioned in contributing guidelines.

**Review by Avdhesh-Varshney — CHANGES_REQUESTED** [review]
@Hacxmr directory name should be `Password-Generator` in kebab-case letters.

**Comment by Hacxmr** [comment]
@Avdhesh-Varshney I have made changes based on the requirement. Can you check and suggest any further modifications are needed

File: Vanilla-JS-Projects/Intermediate/Password-Generator/Readme.md [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+You will see the "Random Password Generator" title, a paragraph indicating the generated password, and a button labeled "Generate Password."
+Generate Password:
+
+Click the "Generate Password" button, and you should see a randomly generated password displayed in the paragraph below.</p>
+</div>
+
+<!-- -------------------------------------------------------------------------------------------------------------- -->
+
+<h3 align='center'>Screenshots 📸</h3>
+![Alt text](Password-Generator.webp)
```
Comment (Avdhesh-Varshney) [inline Vanilla-JS-Projects/Intermediate/Password-Generator/Readme.md:85]: image

@Hacxmr Screenshot is not shown in readme file.
- You have to upload the pic on comment section, image link will be obtained for you.
- As i have done for the above pic `image`
- Then just copy that link and paste it.

**Review by Avdhesh-Varshney — CHANGES_REQUESTED** [review]

**Comment by Hacxmr** [comment]
Hey! @Avdhesh-Varshney  I have made changes based on the input still, I couldn't get a screenshot image. Kindly check

**Comment by Avdhesh-Varshney** [comment]
@Hacxmr no problem i will do it.
PR Approved 🎉

**Pull request merged**